### ISSUE TITLE
How to deal with multiple 1 value in the ground truth mask when you use Sotfmax cross entropy?

### ISSUE THREAD

**Issue by Superlee506** [RodrigoGantier/Mask_R_CNN_Keypoints#6]
Because the mask is only 1 point set to 1 but after crop, resize, round it may have more than 1 point set to 1 as in [L526-L533](https://github.com/RodrigoGantier/Mask_R_CNN_Keypoints/blob/master/model.py#L526-L533).  The softmax loss can handle the only one value with 1.

**Comment by Superlee506** [comment]
use tf. ceil()
